harness/setup.py

```diff
@@ -28,7 +28,7 @@
         "backoff",
         "certifi",
         "filelock",
-        "google-cloud-storage>=1.20.0",
+        "google-cloud-storage>=1.20.0,<=1.40.0",
```

Comment:
**nit:** is this worth commenting in the line above that 1.40.x+ is breaking `lint-python` or whatever conflicts it's causing? (like the `google-cloud-core<1.4.2` line)